src/parser/parser.ts

```diff
@@ -84,6 +84,32 @@ const getEventName = (eventAttributeString: RMLEventAttributeName): [RMLEventNam
  * };
  * ```
  **/
+function stringifyInitialValue(initialValue: any, attributeName?: string): string {
+	const obj = initialValue as Record<string, any>;
+
+	if (attributeName === 'class') {
+		return Object.entries(obj)
+			.filter(([_, v]) => v)
+			.map(([k]) => k)
+			.join(' ');
+	}
+
+	if (attributeName === 'style') {
+		return Object.entries(obj)
+			.map(([k, v]) => `${k}:${v}`)
+			.join('; ');
+	}
+
+	try {
```

Comment:
I'm not convinced whether we need this bit. Either we know exactly what we are rendering, why and how, otherwise... does the fix work without? If it works, we probably don't need this bit at all...